core/src/main/java/com/linecorp/armeria/client/limit/AbstractConcurrencyLimitingClient.java

```diff
@@ -56,10 +56,9 @@
 
     private static final long DEFAULT_TIMEOUT_MILLIS = 10000L;
 
-    private final int maxConcurrency;
-    private final long timeoutMillis;
     private final AtomicInteger numActiveRequests = new AtomicInteger();
```

Comment:
You need to move this field to `ConcurrencyLimit` as well, so that a user can enforce the concurrency level across multiple `ConcurrencyLimitingClient`s.

Reply:
Thanks for your contribution by the way! :bow: